[report]
fail_under=84

[run]
source=vsutil.py
